types/enums/ProjectTypes.ts

```diff
@@ -119,36 +171,83 @@ export const ProjectTypes = {
+  'alter.surfaces.deck': 'Add a decked area',
   'alter.surfaces.parking': 'Add a driveway or parking area',
+  'alter.surfaces.parking.extend': 'Extend a driveway or parking area',
+  'alter.surfaces.parking.new': 'Add a new driveway or parking area',
+  'alter.surfaces.parking.replace': 'Replace a driveway or parking area',
+  'alter.surfaces.patio': 'Add a patio',
+  'alter.surfaces.patio.extend': 'Extend a patio',
+  'alter.surfaces.patio.new': 'Add a new patio',
+  'alter.surfaces.patio.replace': 'Replace a patio',
   'alter.swimmingPool': 'Install a swimming pool',
   'alter.swimmingPool.indoor': 'Install an indoor swimming pool',
   'alter.swimmingPool.outdoor': 'Install an outdoor swimming pool',
   'alter.trees': 'Changes to trees or hedges',
+  'alter.trees.hedge': 'Changes to hedges',
+  'alter.trees.hedge.letGrow': 'Let hedges grow',
+  'alter.trees.hedge.new': 'New hedges',
+  'alter.trees.hedge.prune': 'Prune hedges',
+  'alter.trees.hedge.remove': 'Remove hedges',
+  'alter.trees.tree': 'Changes to trees',
+  'alter.trees.tree.new': 'New trees',
+  'alter.trees.tree.prune': 'Prune trees',
+  'alter.trees.tree.remove': 'Remove trees',
   changeOfUse: 'Change the use of a building',
   'changeofUse.annexe':
     'Convert part of the property into a  granny flat (residential annexe)',
   'changeOfUse.caravans': 'Use a caravan or mobile home on the property',
+  'changeOfUse.extension': 'Convert an extension',
   'changeOfUse.garage': 'Convert a garage',
   'changeOfUse.let.part': 'Let a part of the property',
   'changeOfUse.let.whole': 'Let the property',
   'changeOfUse.outbuilding':
     'Convert or change the use of an outbuilding (such as a shed, garage or barn)',
   'changeOfUse.part': 'Convert or change the use of part of a building',
-  'changeOfUse.whole.homeToHMO': 'Convert a home to bedsits or a shared home',
   'changeOfUse.whole': 'Change the use of a property',
+  'changeOfUse.whole.homeToHMO': 'Convert a home to bedsits or a shared home',
   'changeOfUse.workFromHome': 'Work from home',
   demolish: 'Demolish a building',
   'demolish.boundary': 'Demolish a fence, gate or boundary wall',
   'demolish.full': 'Demolish a building',
+  'demolish.outbuilding': 'Demolish an outbuilding (such as a garage or barn)',
   'demolish.outbuildings': 'Demolish an outbuilding (such as a garage or barn)',
   'demolish.part': 'Demolish part of a building (such as an extension)',
+  'demolish.part.conservatory': 'Demolish part of a conservatory',
   'demolish.replace': 'Demolish a building and build homes in its place',
   extend: 'Extend a building or add an outbuilding',
+  'extend.balcony': 'Enlarge a balcony',
   'extend.basement': 'Add a basement extension',
   'extend.basement.extend': 'Enlarge a basement',
   'extend.basement.lightwell': 'Add a lightwell',
+  'extend.basement.new': 'Add a new basement extension',
   'extend.front': 'Add a front extension',
+  'extend.outbuilding':
```

Comment:
Services currently have mixed use of `extend.outbuilding` singular and `extend.outbuildings` plural. It's less than ideal to include both, but I'd rather be able to remove project types later when service content is tidied up than fail to validate all possible journeys through live services now.

Reply:
Agreed that whilst we're below v1 this is the most pragmatic approach. 

Removing any fields would be a breaking change in the schema but I think at this stage that's actually relatively insignificant as we'd be the ones affected (and the old ones wouldn't get used).

It might be worth looking into how we can decorate these field as deprecated before removing them - possibly overkill at this stage though if it proves tricky tbh!